# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/chromium/sandbox/win/src/job.h
# Commit: 370c40e49230
# Full Hash: 370c40e4923073a87845b57c1f4851bacd9663da
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-06-12 21:42:35
# Regressor Bug: 1552160
# File Overlap Count: 2
# Description:
#   Bug 1552160 Part 1: Roll-up of chromium sandbox update and mozilla patches to get a running browser. r=jld,aklotz,tjr,bobowen
#   
#   This updates security/sandbox/chromium/ files to chromium commit 84108231f6e6e0772fb9a4643679ce76aa771e67.
#   
#   Existing and new patches applied from security/sandbox/chromium-shim/patches/with_update/ to give a compiling and mostly working browser.
# ==============================================================================

diff -r bd5ecd72f76c -r 370c40e49230 security/sandbox/chromium/sandbox/win/src/job.h
--- a/security/sandbox/chromium/sandbox/win/src/job.h	Wed Jun 12 21:55:59 2019 +1200
+++ b/security/sandbox/chromium/sandbox/win/src/job.h	Wed Jun 12 11:10:48 2019 +0100
@@ -16,7 +16,7 @@
 // Handles the creation of job objects based on a security profile.
 // Sample usage:
 //   Job job;
-//   job.Init(JOB_LOCKDOWN, NULL);  //no job name
+//   job.Init(JOB_LOCKDOWN, nullptr);  //no job name
 //   job.AssignProcessToJob(process_handle);
 class Job {
  public:
@@ -26,7 +26,7 @@
 
   // Initializes and creates the job object. The security of the job is based
   // on the security_level parameter.
-  // job_name can be NULL if the job is unnamed.
+  // job_name can be nullptr if the job is unnamed.
   // If the chosen profile has too many ui restrictions, you can disable some
   // by specifying them in the ui_exceptions parameters.
   // If the function succeeds, the return value is ERROR_SUCCESS. If the
@@ -63,5 +63,4 @@
 
 }  // namespace sandbox
 
-
 #endif  // SANDBOX_SRC_JOB_H_